<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Email: </string>
    </property>
   </widget>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="email_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>User id: </string>
    </property>
   </widget>
   <widget class="QLabel" name="user_id_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="main_screen_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти из аккаунта</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_friend_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить в друзья по коду</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friends_list_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Список друзей</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
